plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt

```diff
@@ -103,6 +106,7 @@ open class HtmlRenderer(
             }
             node.hasStyle(TextStyle.Paragraph) -> p(additionalClasses) { childrenCallback() }
             node.hasStyle(TextStyle.Block) -> div(additionalClasses) { childrenCallback() }
+            node.isAnchorable -> buildAnchor(node.anchor, node.anchorLabel!!, node.sourceSetsFilters) { childrenCallback() }
```

Comment:
Must we have a `!!` here?

Reply:
yes, because without it this doesn't make sense. 
Also this wont produce NPE since `isAnchorable` checks if `anchorLabel` != null